spec/controllers/links_controller_spec.rb

```diff
@@ -3,13 +3,10 @@
 RSpec.describe LinksController, type: :controller do
   let(:user) { create(:user) }
 
-  before(:each) do
-    allow_any_instance_of(ApplicationController).
-      to receive(:current_user).and_return(user)
-  end
+  before(:each) { stub_current_user(user) }
```

Comment:
**Code Quality - Nitpick**
Use method from helper to stub `current_user` since it's reused.